xray/plot/plot.py

```diff
@@ -452,6 +452,8 @@ def newplotfunc(darray, ax=None, xincrease=None, yincrease=None,
             # pcolormesh
             kwargs['extend'] = cmap_params['extend']
             kwargs['levels'] = cmap_params['levels']
+            if 'colors' in kwargs:
```

Comment:
I think so. On the other hand, it looks like we've already put the listed colors functionality of `colors` into `cmap` in #509.

@jhamman what do you think? I think we probably want to move the listed colors functionality shown [here](http://xray.readthedocs.org/en/latest/plotting.html#discrete-colormaps) from `cmap` into `colors`. That would more naturally handle the single color case desired here.